flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/PrometheusPushGatewayReporterFactory.java

```diff
@@ -22,13 +22,38 @@
 
 import java.util.Properties;
 
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.DELETE_ON_SHUTDOWN;
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.GROUPING_KEY;
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.HOST;
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.JOB_NAME;
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.PORT;
+import static org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporterOptions.RANDOM_JOB_NAME_SUFFIX;
+
 /** {@link MetricReporterFactory} for {@link PrometheusPushGatewayReporter}. */
 @InterceptInstantiationViaReflection(
         reporterClassName = "org.apache.flink.metrics.prometheus.PrometheusPushGatewayReporter")
 public class PrometheusPushGatewayReporterFactory implements MetricReporterFactory {
 
     @Override
     public PrometheusPushGatewayReporter createMetricReporter(Properties properties) {
-        return new PrometheusPushGatewayReporter();
+        String hostConfig = properties.getString(HOST.key(), HOST.defaultValue());
+        int portConfig = properties.getInteger(PORT.key(), PORT.defaultValue());
+        String jobNameConfig = properties.getString(JOB_NAME.key(), JOB_NAME.defaultValue())
```

Comment:
```suggestion
        String jobNameConfig = properties.getString(JOB_NAME.key(), JOB_NAME.defaultValue());
```

Reply:
I've been working primarily with Kotlin over the last year or so, semicolons aren't my friends. ;)